src/ContinuityMassElemKernel.C

```diff
@@ -61,7 +61,7 @@ ContinuityMassElemKernel<AlgTraits>::ContinuityMassElemKernel(
   dataPreReqs.add_cvfem_volume_me(meSCV);
 
   // fields and data
-  dataPreReqs.add_gathered_nodal_field(*coordinates_, AlgTraits::nDim_);
+  dataPreReqs.add_coordinates_field(*coordinates_, AlgTraits::nDim_);
```

Comment:
Yes. There is an optional parameter that is defaulted. The exact use is 

```
// Register current coordinates 
dataPreReqs.add_coordinates_field(*coordinates_, AlgTraits::nDim_, CURRENT_COORDINATES);

// Register model coordinates
dataPreReqs.add_coordinates_field(*coordinates_, AlgTraits::nDim_, MODEL_COORDINATES);

dataPreReqs.add_master_element_call(GRAD_OP, CURRENT_COORDINATES);
dataPreReqs.add_master_element_call(GRAD_OP, MODEL_COORDINATES);
```

Registers `coordinates` (both model and current) as well as `grad_op` for both these. Retrieval works like this

```
auto v_dndx_model = scratchViews.get_me_elems(MODEL_COORDINATES).dndx;
```